### PR TITLE
Query Escaping

### PR DESCRIPTION
**Pull request by maxifom**
Query escaping and allow to pass params to query

### CODE REVIEW

File: aiochclient/client.py [hunk 1]
```diff
@@ -101,9 +101,22 @@ def __init__(
         ) as resp:  # type: client.ClientResponse
             return resp.status == 200
 
+    @staticmethod
+    def _prepare_query_params(params={}):
```
Comment (maximdanilchenko) [inline aiochclient/client.py]: Please make it None by default with type Optional[dict]

File: aiochclient/client.py [hunk 2]
```diff
@@ -146,7 +159,7 @@ def __init__(
                     async for line in resp.content:
                         yield rf.new(line)
 
-    async def execute(self, query: str, *args, json: bool = False) -> None:
+    async def execute(self, query: str, *args, json: bool = False, params=None) -> None:
```
Comment (maximdanilchenko) [inline aiochclient/client.py]: Can you update documentation string with some examples how to use params argument?

File: aiochclient/client.py (hunk 1 above)
Comment (maxifom) [reply aiochclient/client.py]: Done

File: aiochclient/client.py (hunk 2 above)
Comment (maxifom) [reply aiochclient/client.py]: Done

**Comment by maxifom** [comment]
Added tests

**Comment by maximdanilchenko** [comment]
Cool! Last request. Could you please squash all those "fix" commits? In fact, you can squash all your commits into one or two - it would be ok.

P.S. If you want to fix Travis CI check run `make format`.

**Comment by maxifom** [comment]
@maximdanilchenko Github can automatically squash all commits into one + merge commit. About Travis make format: I cannot see why it says I splitted 1 line into like 6, so the error seems odd to me.

**Comment by maximdanilchenko** [comment]
@maxifom Thanks!

**Comment by maximdanilchenko** [comment]
Closes #19

**Pull request merged**